src/instrumentsscene/qml/MuseScore/InstrumentsScene/InstrumentsPanel.qml

```diff
@@ -183,6 +183,17 @@ Item {
                     flickable.returnToBounds();
                 }
 
+                function scrollToFocusedItem(focusedIndex) {
+                    let targetScrollPosition = focusedIndex * 38
+                    let visibleAreaEnd = flickable.contentY + flickable.height
+
+                    if (targetScrollPosition + 38 > visibleAreaEnd) {
+                        flickable.contentY = Math.min(targetScrollPosition + 38 - flickable.height, flickable.contentHeight - flickable.height)
```

Comment:
I'd prefer not to repeatedly hard-code `38` into this method - instead I think we should give `instrumentsTreeView` a new property so that we can reuse it:

`readonly property real delegateHeight: 38`

We'll also need to update `TreeViewStyle`:

```
rowDelegate: Item {
    height: instrumentsTreeView.delegateHeight
    width: parent.width
}
```

Reply:
Updated now.